src/components/common/FamilyIntake.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, Select, InputNumber } from 'antd';
+
+function FamilyIntake() {
+  const { TextArea } = Input;
+
+  // Initial Empty Values for form
+  const familyInput = {
+    head_of_household: '',
+    family_members: '',
+    family_size: null,
+    education_level: '',
+    homeless_history: '',
+    employment: '',
+  };
+
+  const [familyInfo, setFamilyInfo] = useState(familyInput);
+
+  // Change Handler used for all fields except Select.
+  const formChanges = event => {
+    const { name, value } = event.target;
+    setFamilyInfo({ ...familyInfo, [name]: value });
+  };
+  // antd's answer to onSubmit
+  const onFinish = () => {
+    console.log(familyInfo);
```

Comment:
Please remove this console log from your code. You don't want them in production in your main branch so please remove them. 

If this function's only function is to log out something, it might be good to remove it as well, otherwise please complete the functionality prior to submission.

Reply:
I've taken out the console log and added a placeholder API. Additionally, I have an alert box that shows the submitted data from the placeholder API. The goal is to demonstrate that the data is being submitted. Once the endpoints are determined, one would simply need to substitute a few lines. Also, I️ have included a console.error() in the .catch in case something goes wrong in the future. Screenshot is attached and I️ am pushing the code momentarily. 
![Screen Shot 2022-01-24 at 3 57 26 AM](https://user-images.githubusercontent.com/70237021/150751869-6d8d2fde-f68f-4790-9c7b-be9ce645191d.png)